packages/cmrjs/cmr-class.js

```diff
@@ -0,0 +1,269 @@
+'use strict';
```

Comment:
This is a result of moving stuff from granule over here in stages probably. cmr.js was originally granule-extract-cmr.js but I needed to pull out the class definition from the index.js after the fact.  I don't have any strong feelings about what these files should be called because they still feel like a collection of functions loosely related to cmr.  

How about I leave it for now and you come back and see what they're named after I finish the ticket?  I will think a bit about the organization as I add functionality.

Reply:
There's some garbage in util.js and the non-garbage might find better homes.  I'll rename cmr-class.js to cmr.js and  take what's currently in cmr.js and figure out if it can be moved into util or something like it.

Reply:
Ok.  ```
cmr-class.js => cmr.js
cmr.js => cmr-utils.js
test-cmr.js => test-cmr-utils.js
```
I'll look again when I start adding functionality to this for umm-json.